scripts/get_cldr_data.py

```diff
@@ -0,0 +1,480 @@
+import requests
```

Comment:
But these are imported in the scripts which won't be used by users, do we still need to add these to requirements?